string s
geometry_msgs/Pose goal